charts/rasa-x/templates/rasa-config-files-configmap.yaml

```diff
@@ -24,6 +24,10 @@ data:
       username: {{ template "rasa-x.psql.username" . }}
       password: ${DB_PASSWORD}
       db: ${DB_DATABASE}
+    {{- if .Values.rasa.trackerStore.query }}
+      query:
+    {{- toYaml .Values.rasa.trackerStore.query | nindent 8 }}
+    {{- end }}
```

Comment:
just wondering why the indentation here is different from `useLoginDatabase` below, but seems it doesn't matter in any case since the actual input code is the same indentation